### PR TITLE
Sprint 1 - PR develop to master

### PR DESCRIPTION
**Pull request by MarineMontailler**
This is the first PR to master, after sprint 1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.11",
```
Comment (bugsfunny) [inline package.json:8]: it seems a lot of lib just for icons

File: src/containers/DisplaySearch.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <>
+      <hr></hr>
+      <link
+        rel="stylesheet"
+        href="https://unpkg.com/purecss@2.0.3/build/pure-min.css"
+        integrity="sha384-cg6SkqEOCV1NbJoCu11+bm0NvBRc8IYLRGXkmNrqUBfTjmMYwNKPWBTIKyw9mHNJ"
+        crossOrigin="anonymous"
+      ></link>
+      <div>
+        <p className="search__title">Let's find cocktail</p>
```
Comment (bugsfunny) [inline src/containers/DisplaySearch.js]: it seems you use lot of className in your project, i think you can do more component

File: src/components/MainPage/Search.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  width:100%;
+  margin-top:20px;
+}
+
+button {
+margin-top:10px;
+  margin-right:2px;
+}
+
+.button-secondary {
```
Comment (bugsfunny) [inline src/components/MainPage/Search.js:26]: it seems you use lot of className in your project, i think you can do more component

File: src/containers/DisplaySearch.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <div className="searchBar">
+          <input className="search__input" type="text" placeholder="Search" />
+          <i className="fa fa-2x fa-plus" aria-hidden="true"></i>
+        </div>
+      </div>
+
+      <div className="searchTags">
```
Comment (bugsfunny) [inline src/containers/DisplaySearch.js]: it seems you use lot of className in your project, i think you can do more component

File: src/containers/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Navbar = () => {
+  return (
+    <>
+      <i className="fa fa-2x fa-refresh" aria-hidden="true"></i>
```
Comment (bugsfunny) [inline src/containers/Navbar.js:7]: I think you can replace fa with component (use a lib or svg icon...)

File: src/containers/RandomCocktails.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import axios from "axios";
+
+class RandomCocktail extends React.Component {
```
Comment (bugsfunny) [inline src/containers/RandomCocktails.js]: I think you can change it with functional component

File: src/containers/RandomCocktails.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    };
+    this.getCocktail = this.getCocktail.bind(this);
+  }
+
+  componentDidMount() {
+    this.getCocktail();
+  }
+
+  getCocktail() {
+    axios
```
Comment (bugsfunny) [inline src/containers/RandomCocktails.js:11]: please extract axios logic in other file

File: src/containers/Navbar.js (hunk 5 above)
Comment (MarineMontailler) [reply src/containers/Navbar.js:7]: ok done

File: src/containers/RandomCocktails.js (hunk 6 above)
Comment (MarineMontailler) [reply src/containers/RandomCocktails.js]: ok done

File: package.json (hunk 1 above)
Comment (MarineMontailler) [reply package.json:8]: ok done

File: src/containers/DisplaySearch.js (hunk 2 above)
Comment (MarineMontailler) [reply src/containers/DisplaySearch.js]: in progress for the next sprint

File: src/components/MainPage/Search.js (hunk 3 above)
Comment (MarineMontailler) [reply src/components/MainPage/Search.js:26]: in progress for the next sprint

File: src/containers/DisplaySearch.js (hunk 4 above)
Comment (MarineMontailler) [reply src/containers/DisplaySearch.js]: in progress for the next sprint

File: src/containers/RandomCocktails.js (hunk 7 above)
Comment (MarineMontailler) [reply src/containers/RandomCocktails.js:11]: in progress for the next sprint

**Pull request closed without merging**